### PR TITLE
Advance Character Search integration

### LINKED ISSUE #22: Advanced Character Search

### PR DESCRIPTION
**Pull request by Vasolik**
Along with https://github.com/Vasolik/CK3ACS/commit/0b2938b82fc08ba7ee64f25cbe62d5e93a080ab7 change, this would be all which is needed for Advance Character Search mod. closes #22 

ACS will have this mod as a prerequisite when this mod becomes public. The load order will be irrelevant.

### CODE REVIEW

**Comment by Vasolik** [comment]
Closes #22

File: gui/hud.gui [hunk 1]
```diff
@@ -1221,6 +1233,11 @@ icon = {
 					onclick = "[ToggleEncyclopedia]"
 					shortcut = "encyclopedia"
 					using = tooltip_ne
+					visible = "[Not ( GetScriptedGui('acs_is_loaded').IsShown(GuiScope.End) ) ]"
```
Comment (terrapass) [inline gui/hud.gui]: The space after `Not` seems to break this line, you should remove it, i.e.:
```suggestion
					visible = "[Not( GetScriptedGui('acs_is_loaded').IsShown(GuiScope.End) ) ]"
```

Comment (Vasolik) [reply gui/hud.gui]: when a space is part of the syntax. Fixed

**Comment by Gr770** [comment]
Did testing work for this?

**Pull request merged**